### PR TITLE
Add RBAC missing for crs and ingresses

### PR DESCRIPTION
**Pull request by pipo02mix**
It complaints in the logs, although we are not using them

### CODE REVIEW

**Review by tomahawk28 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project's packages adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [v0.3.2]
+
+### Changed
+
+- Add missing extra RBAC resources missing.
```
Comment (rossf7) [inline CHANGELOG.md]:
```suggestion
- Add missing RBAC resources to cluster role.
```

**Review by rossf7 — APPROVED** [review]
Good catch👍 LGTM

**Comment by MarcelMue** [comment]
For my sanity:
Is this now included if cluster-operator points to `0-3-stable`? I think so, right?
https://github.com/giantswarm/cluster-operator/blob/master/pkg/v17/key/key.go#L92

**Comment by pipo02mix** [comment]
Yes :)

**Pull request merged**